logger/level_logger.go

```diff
@@ -0,0 +1,14 @@
+package logger
+
+import (
+	"context"
+)
+
+type LevelLogger struct {
+	SimpleLogger
```

Comment:
just add this to `SimpleLogger` - we expect downstream dependencies to implement their own logger, but it's fine to add level understanding to `SimpleLogger`

Reply:
Please review, but now the default level should be Info, since it maps to int 0